fission/main.go

```diff
@@ -108,7 +108,7 @@ func main() {
 	specSaveFlag := cli.BoolFlag{Name: "spec", Usage: "Save to the spec directory instead of creating on cluster"}
 
 	// trigger method and url flags (used in function and route CLIs)
-	htMethodFlag := cli.StringFlag{Name: "method", Usage: "HTTP Method: GET|POST|PUT|DELETE|HEAD; defaults to GET"}
+	htMethodFlag := cli.StringFlag{Name: "method", Value: "GET", Usage: "HTTP Method: GET|POST|PUT|DELETE|HEAD"}
```

Comment:
@life1347 Since we are removing the default value message, is there any way for the user to know what the defaults are?

Reply:
If we assign value to `Value `  fields, the cli package will print default value at the end of command description automatically like following.
```
--method value               HTTP Method: GET|POST|PUT|DELETE|HEAD (default: "GET")
--executortype value         Executor type for execution; one of 'poolmgr', 'newdeploy' (default: "poolmgr")
--targetcpu value            Target average CPU usage percentage across pods for scaling (default: 80)

```

Reply:
Great!